external/storm-druid/src/main/java/org/apache/storm/druid/trident/DruidBeamStateUpdater.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.storm.druid.trident;
+
+import org.apache.storm.trident.operation.TridentCollector;
+import org.apache.storm.trident.state.BaseStateUpdater;
+import org.apache.storm.trident.tuple.TridentTuple;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class DruidBeamStateUpdater<E> extends BaseStateUpdater<DruidBeamState<E>> {
+
+    @Override
+    public void updateState(DruidBeamState<E> state, List<TridentTuple> tuples, TridentCollector collector) {
+        List<E> list = new LinkedList<>();
+
+        for (TridentTuple tuple: tuples) {
+            list.add((E)tuple.getValue(0));
```

Comment:
same as above its not clear why are we only picking the first value in the tuple.